### PR TITLE
Fix aarch64 target

### LINKED ISSUE #528: Bad CPU type in executable running macOS on an m1
I am having trouble executing StyLua from the pre-built releases. I tried both [stylua-macos-aarch64](https://github.com/JohnnyMorganz/StyLua/releases/download/v0.14.2/stylua-macos-aarch64.zip) and [stylua-macos](https://github.com/JohnnyMorganz/StyLua/releases/download/v0.14.2/stylua-macos.zip). I assumed `aarch64` was for Apple Silicon but maybe I'm misunderstanding.

```
$ ./stylua example.lua

exec: Failed to execute process: Bad CPU type in executable.
```

My usual method of installing StyLua is `brew install stylua` which works as expected, however there are some tools that automatically manage the installation of formatters and they tend to use the release binaries.

Appreciate any help :)

### PR DESCRIPTION
**Pull request by mvllow**
According to https://github.com/rust-lang/rust/issues/73908 this should be supported. Not sure of an easy way to test the workflow but let me know if there's anything else that's needed :)

Closes #528

### CODE REVIEW

**Review by JohnnyMorganz — APPROVED** [review]
Do you mind adding a quick sentence under "Fixed" in the CHANGELOG?

**Comment by mvllow** [comment]
Added to the unreleased section as I assume this will require a version bump but happy to change if needed!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed aarch64 target in release workflow ([#528](https://github.com/JohnnyMorganz/StyLua/issues/528))
```
Comment (JohnnyMorganz) [inline CHANGELOG.md:11]:
```suggestion
- Fixed macOS aarch64 target in release workflow ([#528](https://github.com/JohnnyMorganz/StyLua/issues/528))
```

**Review by JohnnyMorganz — APPROVED** [review]
Thank you!

**Pull request merged**